<p>copyright of company is needed</p>
<h2>Noncompliant Code Example</h2>
<pre>
    package.......
</pre>
<h2>Compliant Solution</h2>
<pre>
  /**
   *<!-- 添加内容：@(#)<CopyRightRule> -->
   *
   * copyright of kingdee
   *
   */
</pre>
