<p>the kd.bos.threads.ThreadPools must to be used</p>
<h2>Noncompliant Code Example</h2>
<pre>
  java.util.concurrent.ExecutorService;
  java.util.concurrent.Executors;
  java.util.concurrent.ThreadPoolExecutor;
  jdk ThreadPool is not allowed.......
</pre>
<h2>Compliant Solution</h2>
<pre>
  using kd.bos.threads.ThreadPools to deal with the problem.
</pre>
